python_modules/dagster/dagster/_core/definitions/load_assets_from_modules.py

```diff
@@ -21,18 +32,13 @@
 from .source_asset import SourceAsset
 
 
-def _find_assets_in_module(
```

Comment:
Is there some fancy TypeVar thing we can do here? We're accepting types, and yielding instances of those types

Reply:
@smackesey might know?